docs/how-to-guides.md

```diff
@@ -32,6 +32,199 @@ Everything about test data generation and its adjustable parameters can
+     # experiment tests
+     - /ops/exptests@ops.exptests.config.tests: exptests_default.yaml
+     # experiment locations
+     - shared/locations@globals: exp_locations.yaml
+     - _self_
+   
+   hydra:
+     searchpath:
+       - file://configs
+   
+   globals:
+     exp_name: FlowersClsBinary
+     exp_prefix: FLOWERSCLB
+     data_location:
+       uri: ${oc.env:DATA_URI,./data}/kaggle_flowers
+   ```
+
+5. Configure training operation
+   1. We navigate to `configs/ops/train` and create `op_train_cls_binary_flowers.yaml`.
+   2. We can use the content of `op_train_cls_binary_flowers.yaml` as a starting point 
+   and change the definition where the training data is stored, what the image size
+   is and which classes we use as labels. For the time being, we will use the
+   OwnMobileNetModel model of Tensorflow and do not implement our own model definition.
+   ```yaml
+   defaults:
+     - op_train_base.yaml@_here_
+     - /shared/datasets@data_train: dataset_kaggle_flowers.yaml
+     - /shared/datasets@data_validation: dataset_kaggle_flowers.yaml
+     - _self_
+   train_params:
+     epochs: 5
+   data_description:
+     _target_: niceml.data.datadescriptions.clsdatadescription.ClsDataDescription
+     classes:
+     - "daisy"
+     - "roses"
+     use_binary: true
+     target_size:
+       _target_: niceml.utilities.imagesize.ImageSize
+       width: 180
+       height: 180
+   
+   data_train:
+     datainfo_listing:
+       sub_dir: train
+     set_name: train
+     shuffle: true
+   data_validation:
+     datainfo_listing:
+       sub_dir: validation
+     set_name: validation
+   model:
+     _target_: niceml.dlframeworks.tensorflow.models.mobilenet.OwnMobileNetModel
+     final_activation: sigmoid
+   model_load_custom_objects:
+     _target_: niceml.mlcomponents.modelcompiler.modelcustomloadobjects.ModelCustomLoadObjects
+   learner:
+     _target_: niceml.dlframeworks.tensorflow.learners.defaultlearner.DefaultLearner
+     model_compiler:
+       _target_: niceml.dlframeworks.tensorflow.modelcompiler.defaultmodelcompiler.DefaultModelCompiler
+       loss: binary_crossentropy
+       # Optimizer used in the experiment
+       optimizer:
+         _target_: tensorflow.keras.optimizers.Adam
+         learning_rate: 0.0001
+       metrics: [ "accuracy" ]   
+   ```
+
+   4. You might have noticed, that we do not give a plain directory definition to tell
+   niceml where the training images are stored. Instead, we define the datasets within
+   a configuration yaml in `configs/shared/datasets/dataset_kaggle_flowers.yaml`. For now,
+   we will use as the same configuration as in `dataset_cls_test.yaml`
+6. Configure prediction operation
+   1. We create `configs/ops/prediction/op_prediction_cls_binary_flowers.yaml` and use
+   `op_prediction_cls.yaml` as a template.
+   2. Here, we only have to change the definition of the training, validation and test datasets with
```

Comment:
The yaml file always uses the term 'train' or 'train_eval' instead of 'training'. Please use the same terms in the tutorial/documentation to prevent confusion. Please change this in all occurences of 'training', where necessary.